src/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
We already have one in the root, is there a specific reason why you are adding it here as well?

Reply:
I presume this is to include it in the PyPi package?

If so, can we copy it as part of the build to avoid having 2 licenses?

Reply:
I removed it - don't see it being used anymore.